### PR TITLE
Update branding logos

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: Amrish Kushwaha <askmaurya48@gmail.com>

This commit will
- add director-online, director-onprem and openebs-enterprise-platform
logos
- update openebs, litmus and mayadata logos
- add public-resources director which will contains all of zip files of
the logos

### CODE REVIEW

**Comment by IsAmrish** [comment]
logos-one

logos-two

logos-three

File: docs/foundation/logos.mdx [hunk 1]
```diff
@@ -2,53 +2,102 @@
...
+import OpenEBSDark from "../../resources/logos/openebs/openebs-dark.svg";
+
+import LitumsLight from "../../resources/logos/litmus/litmus-white.svg";
+import LitumsDark from "../../resources/logos/litmus/litmus-dark.svg";
 
 # Logos
+
+#### Notes\*
+
+> When you download any of logos, you will get PNG and SVG version of that logo in colored, white and (grey/black) formats.
```
Comment (visualapps) [inline docs/foundation/logos.mdx]: Each download includes all approved formats and color options.

Comment (IsAmrish) [reply docs/foundation/logos.mdx]: or
On downloading any of the logos, you will get PNG and SVG version of that logo in (colored, white and grey/black) formats.

Comment (prasannatk) [reply docs/foundation/logos.mdx]: @IsAmrish Each download includes all approved formats (PNG,SVG) and color options (color,white, grey,black)

**Comment by IsAmrish** [comment]
@prasannatk @visualapps 

logos-final

**Comment by IsAmrish** [comment]
Added padding for x-direction in card
logos-final-final

File: docs/style.scss [hunk 2]
```diff
@@ -16,38 +16,43 @@
...
   font-size: 14px !important;
 }
+
 .logobox {
   border: 1px solid $gray-200;
-  border-radius:4px;
+  border-radius: 4px;
   overflow: hidden;
 }
 .logobox-header {
```
Comment (visualapps) [inline docs/style.scss]: add align-items: center

File: docs/style.scss [hunk 3]
```diff
@@ -16,38 +16,43 @@
...
   display: flex;
   flex-grow: row;
   justify-content: center;
   background: $gray-100;
+  .logobox-header-title {
+    color: $gray-900;
+    font-size: 12px !important;
```
Comment (visualapps) [inline docs/style.scss]: 14px is base font-size 12 is looking small here

File: docs/foundation/js/logobox.js [hunk 4]
```diff
@@ -1,45 +1,50 @@
...
+    console.log("download", this.props.download);
+    window.open(this.props.download);
   }
   render() {
-    return(
-      <div className="logobox">
+    return (
+      <div className={`logobox h-100 ${this.state.theme}`}>
         <div className="logobox-header">
+          <div className="my-auto mr-auto ml-3">
```
Comment (visualapps) [inline docs/foundation/js/logobox.js]: `my-auto mr-auto ml-3 ` not require b/c div inside the flex and flex items are center aligned.

File: docs/style.scss (hunk 3 above)
Comment (visualapps) [reply docs/style.scss]: padding-left: 12px or whatever you add here.

**Comment by IsAmrish** [comment]
final-review

**Comment by IsAmrish** [comment]
@visualapps @Astha2111 @kalyantt @AVRahul 

update new logos 

mayadata-new-logos

**Comment by IsAmrish** [comment]
added openebs director logo and update old logos

de-code-new-logos

**Pull request merged**